src/metrics/mod.rs

```diff
@@ -459,7 +467,7 @@ pub async fn middleware(
                 "cache_status": resp_meta.cache_status,
                 "cache_status_nginx": cache_status_str,
                 "cache_bypass_reason": resp_meta.cache_bypass_reason,
-                "upstream": req_meta.backend.unwrap_or_default(),
+                "backend_host": req_meta.backend.unwrap_or_default(),
```

Comment:
I meant to use the `backend_host` variable here instead of `req_meta.backend.unwrap_or_default()` , not rename the field :)

Reply:
could You plz make the respective commit

Reply:
It's your PR :) You have the `backend_host` variable that you've stripped of port. Just use it everywhere in the logging (in Clickhouse, Vector and normal logging) instead of unparsed `req_meta.backend`

Reply:
> It's your PR :)

I'm not greedy ;) There is a way to offer a code suggestion in the UI, just is it next time to avoid ambiguity. Pushed changes, have a look plz.